packages/api-bindings/src/graphql/utils/profile.ts

```diff
@@ -16,56 +17,71 @@ export type SupportedFollowModuleSettingsFragment =
 
 export type ProfileFieldsFragmentWithSupportedFollowModule = Overwrite<
   ProfileFieldsFragment,
-  { followModule: SupportedFollowModuleSettingsFragment }
+  { __followModule: SupportedFollowModuleSettingsFragment }
 >;
 
 export type ProfileFieldsFragmentWithRevertFollowModule = Overwrite<
   ProfileFieldsFragment,
-  { followModule: RevertFollowModuleSettingsFragment }
+  { __followModule: RevertFollowModuleSettingsFragment }
 >;
 
 export type ProfileFieldsFragmentWithFeeFollowModule = Overwrite<
   ProfileFieldsFragment,
-  { followModule: FeeFollowModuleSettingsFragment }
+  { __followModule: FeeFollowModuleSettingsFragment }
+>;
+
+export type ProfileFieldsFragmentWithUnknownFollowModule = Overwrite<
+  ProfileFieldsFragment,
+  { __followModule: UnknownFollowModuleSettingsFragment }
 >;
 
 export function isProfileFieldsFragmentWithSupportedFollowModule(
   profile: ProfileFieldsFragment,
 ): profile is ProfileFieldsFragmentWithSupportedFollowModule {
-  const followPolicyType = getFollowPolicyTypeFromProfileFieldsFragment(profile);
-  return followPolicyType !== null;
+  const followPolicyType = getFollowPolicyTypeFromProfileFieldsFragment(profile.__followModule);
+  return followPolicyType !== FollowPolicyType.UNKNOWN;
 }
 
 export function isProfileFieldsFragmentWithRevertFollowModule(
```

Comment:
Same here, helper not used in this codebase nor in this PR. `profile.followPolicy.type === FollowPolicyType.NO_ONE` is pretty good for UI checks.